src/client-configuration.ts

```diff
@@ -6,6 +8,11 @@ export interface ClientConfiguration {
    * The {@link URL} of Fauna to call. See {@link endpoints} for some default options.
    */
   endpoint: URL;
+  /**
+   * Determines the encoded format expected for the query `arguments` field, and
+   * the `data` field of a successful response.
+   */
+  format: ValueFormat;
```

Comment:
The default value provided allows the user to not have to define it. But since it is safe to leave out the header, then we can do just that. The additional documentation is also very nice.

Reply:
Done